### ISSUE TITLE
Migrate from module.exports to export const

### ISSUE THREAD

**Issue by reubenae** [intuit/user-data-for-fraud-prevention#9]
At the moment we use module.exports all over the repo. It would be better to have ES6 style `export const ...` instead.

A stack overflow on this topic https://stackoverflow.com/questions/40294870/module-exports-vs-export-default-in-node-js-and-es6 likely this will require some babel configuration.

This can be tested with using yarn-link locally.

**Comment by christyjacob4** [comment]
Hey @reubenae  I'd like to take this up

**Comment by reubenae** [comment]
Good point. This issue could have been written better, the specific location I'd like to remove (if possible) is here: https://github.com/intuit/user-data-for-fraud-prevention/blob/master/src/js/index.js#L6

I will amend the description of the issue, thanks for taking a look!

**Comment by jitinmaher** [comment]
Hi @reubenae , can I take this up?

**Comment by reubenae** [comment]
@jitinmaher you are free to submit a PR, and we appreciate the contribution. We do not allow people to "claim" issues in advance as its too tricky to manage.

Please make sure to test this with the demo project listed in the README.md using `yarn link` or `npm link`

**Comment by reubenae** [comment]
We're not sure here https://github.com/intuit/user-data-for-fraud-prevention/pull/45 - this needs more research done, so I recommend not working on this issue yet (unless you have a really good understanding of this sort of thing and can bring the repo maintainers up to speed!)
